frontend/src/components/auth/UpdateProfileForm.vue

```diff
@@ -0,0 +1,207 @@
+                            type="email"
+                            :rules="emailRules"
+                            required
+                        />
+                        <VSubheader class="edit-form-label">
+                            Password
+                        </VSubheader>
+                        <VTextField
+                            name="input-10-1"
+                            class="edit-form-input"
+                            v-model="editUser.password"
+                            :append-icon="passwordVisibility"
+                            solo
+                            :type="passwordType"
+                            counter
+                            :rules="passwordRules"
+                            required
+                            @click:append="showPassword = !showPassword"
+                        />
+                        <VSubheader class="edit-form-label">
+                            Repeat password
+                        </VSubheader>
+                        <VTextField
+                            name="input-10-1"
+                            class="edit-form-input"
+                            v-model="confirmPassword"
+                            :append-icon="confirmPasswordVisibility"
+                            solo
+                            :type="confirmPasswordType"
+                            counter
+                            :rules="confirmPasswordRules"
+                            required
+                            @click:append="showConfirmPassword = !showConfirmPassword"
+                        />
+                    </VForm>
+                </VCardText>
+                <VCardActions>
+                    <VBtn
+                        @click="onSave"
+                        class="editUser-form-button mt-3"
+                        color="#3C57DE"
+                    >
+                        Save
+                    </VBtn>
+                </VCardActions>
+            </VContainer>
+        </VFlex>
+    </VContent>
+</template>
+
+<script>
+    import {mapGetters, mapActions} from 'vuex';
+    import {UPDATE_USER} from '@/store/modules/auth/types/actions';
+    import {GET_AUTHENTICATED_USER} from '@/store/modules/auth/types/getters';
+
+    export default {
+        data() {
+            return {
+                editUser: {
+                    name: '',
+                    email: '',
+                    password: '',
+                },
+                confirmPassword: '',
+                valid: false,
+                showPassword: false,
+                showConfirmPassword: false,
+                nameRules: [
+                    v => !!v || 'Field full name is required',
+                    v => (v && v.length >= 3) || 'Enter the correct information'
+                ],
+                emailRules: [
+                    v => !!v || 'E-mail is required',
+                    v => /.+@.+\..+/.test(v) || 'E-mail must be valid',
```

Comment:
Add validation service with function like 'validateEmail' and use it here, on login form and registration form